File: 3731 - Find Missing Elements/Method 1/solution.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findMissingElements = function(nums) {
    let list = [];
    let minNum = Number.MAX_SAFE_INTEGER;
let maxNum = Number.MIN_SAFE_INTEGER;
    for(const num of nums) {
        minNum = Math.min(minNum, num);
        maxNum = Math.max(maxNum, num);
    }
    for(let i = minNum; i<=maxNum; i++) {
        let found = false;
        for(let k = 0; k<nums.length; k++) {
            if(i==nums[k]) {
                found = true;
                break;
            }
        }
        if(!found) list.push(i);
    }
    return list;
};

File: 3731 - Find Missing Elements/Method 1/solution.java
class Solution {
    public List<Integer> findMissingElements(int[] nums) {
        List<Integer> list = new ArrayList<>();
        int min = Integer.MAX_VALUE;
        int max = Integer.MIN_VALUE;
        for(int num : nums) {
            min = Math.min(min, num);
            max = Math.max(max, num);
        }
        for(int i = min; i<=max; i++) {
            boolean found = false;
            for(int k = 0; k<nums.length; k++) {
                if(i==nums[k]) {
                    found = true;
                    break;
                }
            }
            if(!found) list.add(i);
        }
        return list;
    }
}